erica/application/grundsteuer/grundsteuer_input_eigentuemer.py

```diff
@@ -32,18 +32,18 @@ class Adresse(CamelCaseInput):
     ort: str
 
 
-class Vertreter(CamelCaseInput):
+class Vertreter(CamelCaseModel):
     name: Name
     adresse: Adresse
     telefonnummer: Optional[str]
 
 
-class Anteil(CamelCaseInput):
+class Anteil(CamelCaseModel):
     zaehler: str
     nenner: str
 
 
-class Person(CamelCaseInput):
+class Person(CamelCaseModel):
     persoenlicheAngaben: PersoenlicheAngaben
```

Comment:
Also snake_case here right?

Reply:
Yes, we could change that @frederike-ramin ?

Reply:
I think so, yes. Let's not change the corresponding element in the Grundsteuer test data for the integration_tests_queue.py, though. This way, we can make sure it still works with the way Grundsteuer webapp is sending the data.